docs/RUP_Architecture.md

```diff
@@ -56,7 +56,17 @@ Please also see this [introduction to the technologies](https://fibo952390745.wo
 
 ### Architectural Goals and Constraints
 
-[This section describes the software requirements and objectives that have some significant impact on the architecture, for example, safety, security, privacy, use of an off-the-shelf product, portability, distribution, and reuse. It also captures the special constraints that may apply: design and implementation strategy, development tools, team structure, schedule, legacy code, and so on.]
+#### Architectural Goals
+The architectural goals of the FiBO app are to provide users with a comprehensive platform for learning and managing personal finances. The app should be user-friendly, with an intuitive interface that enables users to easily access their personal financial information and tools. It should also be secure, with robust data encryption and authentication protocols to protect user data.
+
+#### Architectural Constraints
+The architectural constraints of FiBo include technical limitations, such as various screen sizes. There are also operational constraints such as time and budget limitations as well as a team of developers with a big variety of experiences.
```

Comment:
- Which sizes ? 
- Is it possible to name the sizes ?
- What exactly do you mean with budget limitations ? I didn't know that we had a budget :D